src/components/cms-components/SanityComponentsRenderer.astro

```diff
@@ -19,44 +20,88 @@ interface Props {
 const { content } = Astro.props;
 ---
 
-<div class="flex flex-col gap-y-20">
-  {
-    content?.map((content) => {
-      switch (content._type) {
-        case "cardGrid": {
-          return <CardGrid cardGrid={content} />;
-        }
-        case "servicesCardList": {
-          return <ServiceCardList serviceCardList={content} />;
-        }
-        case "callout": {
-          return <Callout callout={content} />;
-        }
-        case "longFormText": {
-          return <LongFormText longFormText={content} />;
-        }
-        case "tabs": {
-          return <Tabs tabsContent={content} />;
-        }
-        case "highlight": {
-          return <HighlightBlock highlight={content} />;
-        }
-        case "workCardList": {
-          return <WorkCardList workCardList={content} />;
-        }
-        case "testimonial": {
-          return <Quote testimonial={content} />;
-        }
-        case "blogHighlight": {
-          return <BlogHighlight blogPost={content} />;
-        }
-        case "blogsList": {
-          return <BlogsList blogsList={content} />;
-        }
-        case "codeEmbed": {
-          return <CodeEmbed code={content} />;
-        }
-      }
-    })
-  }
-</div>
+{
+  content?.map((content) => {
```

Comment:
I feel like instead of adding a wrapper to all components you should rather update the few sanity components which are essentially wrappers already? I'll properly review later ubt this seems a bit too complex for little gain

Reply:
With the wrapper we can adjust the padding easily. If we apply it to every component individually we have to adjust it in 20 places everytime we adjust the padding or anything else.